railties/lib/rails/commands.rb

```diff
@@ -8,6 +8,7 @@
   "db" => "dbconsole",
   "r"  => "runner",
   "t"  => "test",
+  "k"  => "rake"
```

Comment:
Don't think querying rake version and help info is a common enough command to warrant an alias. Let's take it out.